lumicks/pylake/tests/test_file.py

```diff
@@ -76,6 +76,25 @@ def test_properties(h5_file):
     assert f.fdcurves == {}
 
 
+def test_groups(h5_file):
+    f = pylake.File.from_h5py(h5_file)
+    if f.format_version == 1:
+        assert set(f.keys()) == set(f.h5.keys())
+        assert str(f["Force HF"]) == "Group (members: Force 1x, Force 1y)"
+
+        for x in range(0, 2):
+            t = []
+            for name in f:
+                t.append(name)
```

Comment:
This can be simplified to `t = [name for name in f]` or even just `list(f)`.

Reply:
Ah, yes, I'll use the list comprehension.